server/routes/avatar/utils.js

```diff
@@ -69,15 +69,15 @@ export function userCanAccessAvatar({ headers = {}, query = {} }) {
 
 const getFirstLetter = (name) => name.replace(/[^A-Za-z0-9]/g, '').substr(0, 1).toUpperCase();
 
-export const renderSVGLetters = (username, viewSize = 200) => {
+export const renderSVGLetters = (username, viewSize = 200, avatarColor) => {
```

Comment:
I would create an extra object parameter to pass this kind of information.
```suggestion
export const renderSVGLetters = (username, viewSize = 200, { avatarColor }) => {
```